### PR TITLE
fix: fix unit tests

### PR DESCRIPTION
**Pull request by Erigara**
## Description

Fix failing unit tests.

Checked tests locally to verify if errors are resolved.

```
cargo test --all-features --no-fail-fast --workspace --exclude iroha_executor
```

### CODE REVIEW

File: core/src/smartcontracts/isi/mod.rs [hunk 1]
```diff
@@ -485,23 +482,4 @@ mod tests {
         ));
         Ok(())
     }
-
-    #[test]
-    async fn transaction_signed_by_genesis_account_should_be_rejected() -> Result<()> {
```
Comment (Erigara) [inline core/src/smartcontracts/isi/mod.rs:490]: This check was removed from `accept` function.

Comment (Erigara) [reply core/src/smartcontracts/isi/mod.rs:490]: @mversic any opinion on this one?

Comment (mversic) [reply core/src/smartcontracts/isi/mod.rs:490]: I can't recall why I removed it

Comment (mversic) [reply core/src/smartcontracts/isi/mod.rs:490]: I think it should be returned

Comment (Erigara) [reply core/src/smartcontracts/isi/mod.rs]: returned

**Review by 0x009922 — APPROVED** [review]

**Review by mversic — APPROVED** [review]

**Pull request merged**